From 73ad27b791cfe921fabc846fcea6fbeede2aebdc Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Tue, 15 Sep 2020 11:44:46 +0800
Subject: [PATCH] ARM: dts: rockchip: correct battery info for
 rv1126-bat-ipc-v10

vcc_bat has no voltage divider, so correct it.

Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
Change-Id: Ibb3eb7b32c84414bf47677c422d2cbf3b1ddbac3
---
 arch/arm/boot/dts/rv1126-bat-ipc-v10.dts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts b/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
index 18892ca2c583..fc65695e5e4b 100644
--- a/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
+++ b/arch/arm/boot/dts/rv1126-bat-ipc-v10.dts
@@ -208,8 +208,6 @@
 				   0xCB 0x2F 0x00 0x64 0xA5 0xB5 0x00 0xF8 0x31>;
 		monitor_sec = <5>;
 		virtual_power = <0>;
-		divider_res1 = <200>;
-		divider_res2 = <200>;
 	};
 
 	pcf8563: pcf8563@51 {
-- 
2.35.3

